protocol/communities/community.go

```diff
@@ -233,6 +237,10 @@ func (o *Community) MarshalJSON() ([]byte, error) {
 	}{
 		ID:                          o.ID(),
 		MemberRole:                  o.MemberRole(o.MemberIdentity()),
+		IsAdmin:                     o.IsAdmin(),
+		IsOwner:                     o.IsOwner(),
+		IsControlNode:               o.IsControlNode(),
+		HasAdminPrivileges:          o.IsOwnerOrAdmin(),
```

Comment:
I am wondering if we really need to expose that, can't clients simply check if member role is either owner or admin? I wouldn't consider it as crucial logic duplication among clients.

Reply:
fixed